ui/StatusQ/src/StatusQ/Controls/StatusInput.qml

```diff
@@ -515,13 +523,33 @@ Control {
             onEditingFinished: {
                 root.editingFinished()
             }
+
+            // use the default padding values from StatusBaseInput, but forward any explicitly set values from outside
+            Binding on topPadding {
+                value: root.topPadding
+                when: root.topPadding !== internal.noPadding && root.topPadding !== statusBaseInput.topPadding
+                restoreMode: Binding.RestoreBinding
+            }
+            Binding on bottomPadding {
+                value: root.bottomPadding
+                when: root.bottomPadding !== internal.noPadding && root.bottomPadding !== statusBaseInput.bottomPadding
+                restoreMode: Binding.RestoreBinding
+            }
+            Binding on leftPadding {
+                value: root.leftPadding
+                when: root.leftPadding !== internal.noPadding && root.leftPadding !== statusBaseInput.leftPadding
+                restoreMode: Binding.RestoreBinding
+            }
+            Binding on rightPadding {
+                value: root.rightPadding
+                when: root.rightPadding !== internal.noPadding && root.rightPadding !== statusBaseInput.rightPadding
+                restoreMode: Binding.RestoreBinding
+            }
         }
 
         RowLayout {
             id: bottomRow
-            Layout.topMargin: Theme.halfPadding
             Layout.fillWidth: true
-            Layout.preferredHeight: Math.max(bottomLabelMessageLeft.height, errorMessage.height, bottomLabelMessageLeft.height)
```

Comment:
This was wrongly giving some positive height, even if hidden

Reply:
I agree with your statement, but at that time this line was added to fix the jumping content. See the video below to see how the content moves when hovering over the last fee option.

https://github.com/user-attachments/assets/e3a8dc08-39b9-41d5-8e7f-02c59768dcfb

Reply:
Yeah I see... why is this even an "input"? :laughing: you can't type anything here

Reply:
I tried reverting my changes locally to the previous version and it still does that... so this line is not the culprit; it was likely this way even before

Reply:
> Yeah I see... why is this even an "input"? 😆 you can't type anything here

Ah I see... it's the custom option

Reply:
In action:

https://github.com/user-attachments/assets/0e59f140-3d3a-49d9-bb33-655b706c70d3

Reply:
I'll send a separate PR for this

Reply:
Submitted at https://github.com/status-im/status-desktop/pull/17421